### ISSUE TITLE
Consider uploading to pypi

### ISSUE THREAD

**Issue by jon-chuang** [stanford-futuredata/ColBERT#225]
There are at least 2 dependent projects now: 
- Intel's FastRAG which vendors the repo
- llama_index: https://github.com/jerryjliu/llama_index/pull/6818

It would be nice if users could `pip install colbert-ir` instead of having to install from source.

**Comment by okhat** [comment]
Agreed. We already have a working setup.py so this sounds not too hard

**Comment by jon-chuang** [comment]
Note that `colbert` name is already taken. https://pypi.org/project/Colbert/

Another requirement is to be able to do `pip install colbert[cpu]`

**Comment by thesillypeanut** [comment]
Hello! Is there a timeline for this work by any chance?

**Comment by okhat** [comment]
@krypticmouse @hmoazam @hichewss are on it, among other work they're doing, some roadblocks due to faiss but getting close

**Comment by Alexandre-SCHOEPP** [comment]
> Agreed. We already have a working setup.py so this sounds not too hard

I believe that #268 indicates that the setup.py is not working fully. It is not exposing the .cpp source files, if I'm correct.
